### PR TITLE
update buckifier for new json format and updated macros

### PR DESCRIPTION
**Pull request by likewhatevs**
Differential Revision: D34543573

### CODE REVIEW

File: buckifier/targets_cfg.py [hunk 1]
```diff
@@ -41,6 +41,6 @@
 """
 
 fancy_bench_template = """
-fancy_bench_wrapper(suite_name="{name}", binary_to_bench_to_metric_list_map={bench_config}, slow={slow}, expected_runtime={expected_runtime})
+fancy_bench_wrapper(suite_name="{name}", binary_to_bench_to_metric_list_map={bench_config}, slow={slow}, expected_runtime={expected_runtime}, sl_iterations={sl_iterations}, regression_threshold={regression_threshold})
```
Comment (jay-zhuang) [inline buckifier/targets_cfg.py:44]: Should we set a default value here?
Is there any other parameter we could use?
Is the setting per test or test_suit?

**Review by jay-zhuang — APPROVED** [review]
LGTM

File: buckifier/targets_cfg.py (hunk 1 above)
Comment (likewhatevs) [reply buckifier/targets_cfg.py:44]: The json's contain default parameters, these are per test suite.

Comment (jay-zhuang) [reply buckifier/targets_cfg.py:44]: would be great to have per test config. (maybe with later diff).

**Pull request closed without merging**